adduser.sh

```diff
@@ -8,5 +8,5 @@ else
   else
     jx create user --login=${USER_EMAIL}
   fi
-  jx edit userroles --login=${USER_EMAIL} --role=owner
+  jx edit userroles --login=${USER_EMAIL} --role=owner,jx-view
```

Comment:
@cagiti the error I'm getting requires environment list at the cluster scope, I can see the jx-view role can list environments but it's not a clusterrole, is this going to work ?